### ISSUE TITLE
Null error when calling another mod

### ISSUE THREAD

**Issue by Paddy69** [PORTB/BiggerStacksIssues#23]
crashes on startup, 
looping error then hard crash `[02:42:34] [Worker-Main-14/WARN] [po.bi.BiggerStacks/]: Stack size ruleset is somehow null, using fallback logic. Called from com.minecolonies.coremod.research.GlobalResearch [GlobalResearch.java:737]`
logs below
https://gist.github.com/Paddy69/5b055f2a7a4c794ee1c96afc53e406a6
any recommendations?

**Comment by P3rf3ctXZer0** [comment]
[09.03 22:36:20] [Server] [main/WARN] [po.bi.BiggerStacks/]: Stack size ruleset is somehow null, using fallback logic. Called from com.minecolonies.coremod.research.GlobalResearch [GlobalResearch.java:732]

**Comment by Paddy69** [comment]
right, forget got about this, i found the issue. another one if my devs tried to ban someone by just writeing their name into the banned players file and it caused a java read error. there is no issue with your mod
